src/components/forms/login/LoginCard.js

```diff
@@ -16,30 +17,27 @@ const LoginCard = ({onAlert}) => {
           try {
             let response = await Api.post('/login', Object.fromEntries(data.entries()))
```

Comment:
why not do the object.entries thing and get your data and then give it to the endpoint